eventuals/foreach.h

```diff
@@ -13,7 +14,9 @@ template <typename E, typename F>
 auto Foreach(E e, F f) {
   return std::move(e)
       | Map(std::move(f))
-      | Loop();
+      | Loop()
+            .raises_tuple<
+                ErrorsFromMaybeComposable<decltype(e), void, std::tuple<>>>();
```

Comment:
But this `Loop()` doesn't raise that ...

That will get raised from `E`, so anyone who does `ErrorsFrom` on what gets returned from `Foreach` will have the correct possible errors, but we shouldn't need to add that on `Loop`.